Lib/idlelib/run.py

```diff
@@ -401,17 +401,22 @@ def handle_error(self, request, client_address):
 
 # Pseudofiles for shell-remote communication (also used in pyshell)
 
-class PseudoFile(io.TextIOBase):
+class StdioFile(io.TextIOBase):
 
-    def __init__(self, shell, tags, encoding=None):
+    def __init__(self, shell, tags, encoding='utf-8', errors='strict'):
```

Comment:
In all uses, there is one tag: 'stdin', 'stdout', 'stderr', 
```suggestion
    def __init__(self, shell, tag, encoding='utf-8', errors='strict'):
```

Reply:
Yes, but potentially it can take several tags. I'll left it for other issue.

Reply:
ok